jose/jwt.py

```diff
@@ -167,7 +182,7 @@ def decode(token, key, algorithms=None, options=None, audience=None, issuer=None
     return claims
 
 
-def get_unverified_header(token):
+def get_unverified_header(token: str):
```

Comment:
Could we add a return type hint to this and a few other functions where it is missing?

```suggestion
def get_unverified_header(token: str) -> Dict[str, Any]:
```